js/templates/modals/about/license.html

```diff
@@ -0,0 +1,36 @@
+<div class="contentBox padMd clrP clrBr clrSh3">
+  <div class="flexSpaceBetween">
```

Comment:
See earlier comment on restructuring this, and doing it once instead of repeating it.